docs/user-guides/dashboard-as-code.md

```diff
@@ -66,6 +66,108 @@ If the build is successful, the result can be found in the generated `built` fol
 > [!NOTE]
 > the `-o` (alternatively '--output') flag is optional (the default output format is YAML).
 
+## Getting started with Go SDK
```

Comment:
imho it's weird to have at same level
- get started with cue
- develop dashboards
- get started with go
- develop dashboards

We should have
- cue
  - get started
  - develop dashboards
- go
  - get started
  - develop dashboards

-> maybe worth having 2 different files then. As you prefer.

Reply:
I moved to:
```
## Getting Started With Cue
### Prerequisites
### Develop dashboards

## Getting Started With Go
### Prerequisites
### Develop dashboards
```